### ISSUE TITLE
Badges are gone

### ISSUE THREAD

**Issue by R7010A** [thunderbird/thunderbird-android#6797]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

5.310

### Where did you get the app from?

Google Play

### Android version

12

### Device model

OnePlus Nord 2

### Steps to reproduce

1. Receive Mail
2. No badge is shown

### Expected behavior

K-9 Mail should show badges on the app icon when new emails arrive.

### Actual behavior

K-9 Mail does not show badges for new incoming emails.
Bug exists since update to v5.310
Tried it on several launchers like Nova, Standard-launcher, AIO, Niagara, no problem there. Badges are shown as expected.

**Comment by cketti** [comment]
There is no K-9 Mail version 5.310.

Showing a number on an app icon is not a standard Android feature. K-9 Mail doesn't have special support for third-party launchers supporting this.

However, there is a home screen widget showing the unread count. See https://docs.k9mail.app/en/6.400/misc/widgets/#k-9-unread

**Comment by jpasher-lazor** [comment]
I believe what this issue is reporting is that the "K-9 Unread" widget the docs reference in your link no longer exists. My K-9 just updated to 6.600, and my home screen icons disappeared. When I added the widget back, my only choice was "K-9 Accounts", which doesn't provide the unread count badge.

Maybe it's related to #6563 , but that seems to just move the option to the manifest file. It mentions the widget will be enabled programmatically. How does that happen? Is something needed to trigger that? I tried reinstalling the app to no avail.
